### PR TITLE
Add United Domains USA

### PR DESCRIPTION
**Pull request by seitsu**
Add United Domains USA. Please keep the USA in the title, because there is also a German website (https://www.united-domains.de).

### CODE REVIEW

File: _data/domains.yml [hunk 1]
```diff
@@ -476,6 +476,13 @@ websites:
       tfa: Yes
       software: Yes
       doc: https://uniregistry.help/my-account-en/what-is-two-step-verification/
+      
+    - name: United Domains USA
+      url: https://www.uniteddomains.com/
+      img: uniteddomains-usa.png
+      tfa: Yes
+      software: Yes
+      doc: https://www.uniteddomains.com/twofactorauth/
```
Comment (mxxcon) [inline _data/domains.yml]: Is there a better link that actually explains how to enable 2fa?

Comment (mxxcon) [inline _data/domains.yml]: I think this is a better link to use https://help.uniteddomains.com/hc/en-us/articles/115001458813-How-to-Enable-Disable-Two-Factor-Authentication

**Review by mxxcon — CHANGES_REQUESTED** [review]

File: _data/domains.yml (hunk 1 above)
Comment (seitsu) [reply _data/domains.yml]: Agreed

**Review by Jawshy — COMMENTED** [review]
Thanks for the pull request, @seitsu. 

I'm confused about how the user receives their single-use, time-sensitive code. [This image](https://www.uniteddomains.com/img/tfa_2.jpg) on the [2 Factor Authentication](https://www.uniteddomains.com/twofactorauth/) page suggests an SMS based method; whereas this [help centre (FAQ) article mentions an authenticator app](https://help.uniteddomains.com/hc/en-us/articles/115001458833-About-Recovery-Codes).

**Comment by seitsu** [comment]
Thanks for the note, @Jawshy. It is with an authenticator app, not via SMS. I dropped a note to the product team to change the image to make it clearer.

**Review by Jawshy — APPROVED** [review]
I appreciate the clarification. Approved! 👍

**Review by mxxcon — APPROVED** [review]

**Pull request merged**